### ISSUE TITLE
Don't cluster people by XP when forming teams

### ISSUE THREAD

**Issue by tannerwelsh** [LearnersGuild/echo#344]
**tl;dr** when forming teams (after grouping by votes), don't create teams with the least XP differential across members. instead, create teams of random XP distribution.

this is a fix requested by @shereefb to avoid the situation of having the same people on teams repeatedly. case in point: Harman and Yaseen have been on the same team two times in a row, and their XP is growing at consistent rates, so it is likely that they will be on the same team again.

(in the future, we should have a more sophisticated way to guard against repeated team memberships, but this will do for now and is needed urgently by tomorrow.)
